wordle-web/components/WordList.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <v-bottom-sheet
+    v-model="modelValue"
+    max-height="600"
+    class="pa-2 d-flex flex-wrap justify-center align-center"
+  >
+    <v-card>
+      <v-autocomplete
```

Comment:
Super super sick. I am thinking about implementing this in our website.

Reply:
Thanks! The whole scrolling part really didn't make sense to me so having autocomplete makes it easier for user to navigate.